src/Common/src/TypeSystem/Common/TypeSystemException.cs

```diff
@@ -193,6 +193,10 @@ public static string OwningModule(MethodDesc method)
 
             public static string OwningModule(TypeDesc type)
             {
+#if TYPE_LOADER_IMPLEMENTATION
```

Comment:
@fadimounir It's not okay to ifdef around in the common type system files. We split everything into partial classes so that we don't need ifdefs. If nometadata types can provide an owning module, we should just move the property to DefType. Or there can just be a separate no metadata based type name formatter in the type loader with separate rules. I don't want this to set bad precedents in the common type system.

Cc @davidwrighton